lib/request.js

```diff
@@ -50,7 +50,7 @@ function getTrustProxyFn (tp) {
   }
   if (tp === true) {
     // Support plain true/false
-    return function () { return true }
+    return undefined
```

Comment:
How does this mesh with the comment? What is the history of the comment and the code?

Reply:
For me it looks like the comment refers to the argument of `getTrustProxyFn` but I'm not sure

Reply:
I think that comment is confusing with this change, it could be something like “trust everything”

Reply:
Maybe "Support trusting everything" to match other comments below?